views/pages/partnership-details.html

```diff
@@ -0,0 +1,250 @@
+            <td>{{record.fatherOfPartner2.occupation}}{{#record.fatherOfPartner2.retired}} (retired){{/record.fatherOfPartner2.retired}}</td>
+        </tr>
+        <tr>
+            <th><span class="visuallyhidden" aria-hidden="true">Father of partner 2's </span>Designation</th>
+            <td>{{record.fatherOfPartner2.designation}}</td>
+        </tr>
+        <tr>
+            <th><span class="visuallyhidden" aria-hidden="true">Father of partner 2's </span>Deceased</th>
+            <td>{{#record.fatherOfPartner2.deceased}}Yes{{/record.fatherOfPartner2.deceased}}{{^record.fatherOfPartner2.deceased}}No{{/record.fatherOfPartner2.deceased}}</td>
+        </tr>
+        <tr class="section-head">
+            <th colspan="2">Mother of partner 2</th>
+        </tr>
+        <tr>
+            <th><span class="visuallyhidden" aria-hidden="true">Mother of partner 2's </span>Surname</th>
+            <td>{{record.motherOfPartner2.surname}}</td>
+        </tr>
+        <tr>
+            <th><span class="visuallyhidden" aria-hidden="true">Mother of partner 2's </span>Forename(s)</th>
+            <td>{{record.motherOfPartner2.forenames}}</td>
+        </tr>
+        <tr>
+            <th><span class="visuallyhidden" aria-hidden="true">Mother of partner 2's </span>Occupation</th>
+            <td>{{record.motherOfPartner2.occupation}}{{#record.motherOfPartner2.retired}} (retired){{/record.motherOfPartner2.retired}}</td>
+        </tr>
+        <tr>
+            <th><span class="visuallyhidden" aria-hidden="true">Mother of partner 2's </span>Designation</th>
+            <td>{{record.motherOfPartner2.designation}}</td>
+        </tr>
+        <tr>
+            <th><span class="visuallyhidden" aria-hidden="true">Mother of partner 2's </span>Deceased</th>
+            <td>{{#record.motherOfPartner2.deceased}}Yes{{/record.motherOfPartner2.deceased}}{{^record.motherOfPartner2.deceased}}No{{/record.motherOfPartner2.deceased}}</td>
+        </tr>
+        <tr class="section-head">
+            <th colspan="2">First witness</th>
+        </tr>
+        <tr>
+            <th><span class="visuallyhidden" aria-hidden="true">First witness's </span>Signature</th>
+            <td>{{record.witness1.forename}} {{record.witness1.surname}}</td>
+        </tr>
+        <tr class="section-head">
+            <th colspan="2">Second witness</th>
+        </tr>
+        <tr>
+            <th><span class="visuallyhidden" aria-hidden="true">Second witness's </span>Signature</th>
+            <td>{{record.witness2.forename}} {{record.witness2.surname}}</td>
+        </tr>
+      {{/showAll}}
+      <tr class="section-head">
+        <th colspan="2">Registration</th>
+      </tr>
+      {{#showAll}}
+        <tr>
+            <th>Registrar signature</th>
+            <td>{{record.registrar.signature}}</td>
+        </tr>
+      {{/showAll}}
+      <tr>
+        <th>Date of registration</th>
+        <td>{{record.date}}</td>
```

Comment:
Presumably we should remove this.

Reply:
https://github.com/UKHomeOffice/lev-web/pull/415

Reply:
Yes, it should be removed, although I thought you had a change waiting to do this?

Reply:
I think I'd rather merge this PR, then handle this change separately...